components/ToDoForm.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import Image from "next/image";
+import addButtonCircleActive from "../assets/addButtonCircleActive.svg";
+
+export default function ToDoForm({ onSubmitNewToDoItem, destinationId }) {
+  return (
+    <ToDoListForm
+      onSubmit={(event) => onSubmitNewToDoItem(event, destinationId)}
+      aria-label="add todo form"
+    >
+      <AddToDoButton aria-label="submit">
```

Comment:
submit as an area label does not give any new info. The screenreader already knows that is a button of type="submit" (type submit is the default value for buttons). Use something descriptive that you would write inside the button if you didn't have an image.

Reply:
Changed.